ext/calendar/calendar.c

```diff
@@ -218,7 +218,7 @@ enum { CAL_MONTH_GREGORIAN_SHORT, CAL_MONTH_GREGORIAN_LONG,
 };
 
 /* for heb_number_to_chars */
-static char alef_bet[25] = "0����������������������";
+static char alef_bet[25] = "0\xE0\xE1\xE2\xE3\xE4\xE5\xE6\xE7\xE8\xE9\xEB\xEC\xEE\xF0\xF1\xF2\xF4\xF6\xF7\xF8\xF9\xFA";
```

Comment:
Add comment about used encoding here?

Reply:
Maybe it makes sense to show the UTF-8 encoded hebrew letters in a comment here?

Reply:
Sure, comment with encoding and hebrew characters in current UTF-8 encoded file is useful and coming up... Issue is that Hebrew is written right to left. So the first `\xE0` is not the letter first seen on the left. But it will be better than now for sure.

Reply:
RTL doesn't seem to be an issue here, since `alef_bet` is a lookup table.

Reply:
applied